ci/build.py

```diff
@@ -121,7 +185,7 @@ def build_docker(platform: str, docker_binary: str, registry: str, num_retries:
     # container match permissions of the local user. Same for the group.
     #
     # These variables are used in the docker files to create user and group with these ids.
-    # see: docker/install/ubuntu_adduser.sh
+    # see: dockerfiles/install/ubuntu_adduser.sh
```

Comment:
Let's not move the files in this change for now. We probably would like this change to be merged to release branches as well and we'd need to keep it as crisp as possible.